automation/start_flex_aio_demo.sh

```
@@ -0,0 +1,576 @@
+}
+
+install_docker() {
+    echo "[I] Checking docker"
+
+    docker_exists=$(type docker 2>/dev/null ||:)
+
+    if [[ -z "$docker_exists" ]]; then
+        echo "[W] docker doesn't exist; installing ..."
+
+        # setup docker repository
+        sudo apt remove $(dpkg --get-selections docker.io docker-compose docker-compose-v2 docker-doc podman-docker containerd runc | cut -f1)
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)